scripts/check-udeps.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+#
+# This script makes sure that there is unused dependencies in the tikv
+# dependencies tree. It is suitable to run as part of CI.
+
+cargo install cargo-udeps && cargo udeps
```

Comment:
How about `which cargo-udeps &>/dev/null || cargo install cargo-udeps && cargo udeps`?

Reply:
cargo does not force install udeps if it is already instead.

Reply:
It will access network first, which can be slow sometime.